bowtie/schemas/public-implementations.json

```diff
@@ -0,0 +1,77 @@
+{
+  "title": "Public Implementations",
+  "description": "A Public API providing metadata about implementations of JSON Schema supported by Bowtie",
+
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+
+  "$id": "tag:bowtie.report,2024:models:public-implementations",
```

Comment:
This isn't a model (a model is like "a concept", so e.g. an implementation class' schema). Let's make its ID be similar to the above -- `tag:bowtie.report,2024:api:v1:json-schema-org:implementations`.

It should really also share as much as possible with the definitions -- i.e. rather than redefining all the properties we should share them. That though requires moving some models to a common place, so we can perhaps do that after as well.